content/en/docs/ops/integrations/certmanager/index.md

```diff
@@ -42,7 +60,7 @@ spec:
   ...
 {{< /text >}}
 
-Once we have the certificate created, we should see the secret created in the `istio-system` namespace.
+1. Once we have the certificate created, we should see the secret created in the `istio-system` namespace.
 This can then be referenced in the `tls` config for a Gateway under `credentialName`:
 
 {{< text yaml >}}
```

Comment:
You'll probably need to indent this text block as well so it's under the list item